Source/reactions/Castro_react_util.H

```diff
@@ -0,0 +1,265 @@
+#ifndef CASTRO_REACT_UTIL_H
+#define CASTRO_REACT_UTIL_H
+
+#include <Castro.H>
+#include <burn_type.H>
+// #include <actual_rhs.H>
+
+AMREX_GPU_HOST_DEVICE AMREX_INLINE
+void
+single_zone_react_source(GpuArray<Real, NUM_STATE> const& state,
+                         GpuArray<Real, NUM_STATE>& R,
+                         burn_t burn_state);
+
+AMREX_GPU_HOST_DEVICE AMREX_INLINE
+void 
+numerical_jac(burn_t& state, RArray2D& jac) {
+    return;
```

Comment:
I've implemented this now in Microphysics.  Once that PR is merged, we can hook it in.